pkg/fanal/secret/scanner.go

```diff
@@ -341,6 +341,7 @@ type Match struct {
 func (s *Scanner) Scan(args ScanArgs) types.Secret {
 	// Global allowed paths
 	if s.AllowPath(args.FilePath) {
+		log.Logger.Debugf("found global allowed paths %q", args.FilePath)
```

Comment:
We should clarify it is relevant to secret scanning.

```suggestion
		log.Logger.Debugf("Skipped secret scanning on %q matching allowed paths", args.FilePath)
```

Reply:
yes, sure